modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/file/FileStorage.java

```diff
@@ -322,7 +322,8 @@ public void moveStorage(File oldDirectory, File newDirectory, PodcastManager pod
 							if (fileName.length() < 36) {
 								continue;
 							}
-							PodcastEpisode episode = episodeManager.findByUuid(fileName);
+							//noinspection deprecation
+							PodcastEpisode episode = episodeManager.findByUuidSync(fileName);
```

Comment:
Leaving the synchronous calls in this class since using coroutines in java at our API level is not trivial. It's certainly doable though, so let me know if you'd rather us get rid of these.